activation_service_poc/docs/README.md

```diff
@@ -0,0 +1,128 @@
+
+The architecture of the PoC is as follows:
+
+![design.png](design.png)
+
+The PoC consists of two separate processes:
+* `node service` aka node
+* `smesher service` aka client
+
+The easiest way to understand the setup is to try the PoC yourself. The following instructions will guide you through the process.
+
+
+## Running the PoC
+
+There are two distinct configuration/setup methods possible:
+1. Using locally running node and smesher service
+2. Using remote running node and local smesher service
+
+We will use a Docker Compose based setup for both options. While it's possible to run the PoC without Docker, it's more complicated, so for simplicity we'll use Docker.
+
+Both setups will run two smesher services connecting to the same node. Please note that one smesher service can have multiple post services connected to it, as is currently the case with node and post services. However, in this PoC we're using only one post service per smesher service.
+
+### Using locally running node and smesher service
+
+Please use the `docker-compose-testnet-both-local.yml` file for this setup.
+
+The commands will be:
+```
+docker-compose -f docker-compose-testnet-both-local.yml [...]
+```
+
+Since this will be a locally running node, it will need to sync with the network like all Spacemesh nodes. The testnet network is small, so the node should sync relatively quickly.
+
+
+### Using remote running node and local smesher service
+
+Please use the `docker-compose-testnet-remote-node.yml` file for this setup.
+
+The commands will be:
+```
+docker-compose -f docker-compose-testnet-remote-node.yml [...]
+```
+
+### Starting the setup
+
+1. Clone the repository and switch to the `node-split-poc` branch
+```
+git clone https://github.com/spacemeshos/go-spacemesh.git
+cd go-spacemesh
+git checkout node-split-poc
+```
+2. Change directory to the demo directory
+```
+cd activation_service_poc/demo
+```
+3. You should see the following files in the directory
+```
+docker-compose-testnet-both-local.yml
+docker-compose-testnet-remote-node.yml
+```
+4. Run the docker-compose command according to your chosen setup. For example, to run the setup with locally running node and smesher service, run:
+```
+docker-compose -f docker-compose-testnet-both-local.yml up -d
+```
+
+
+If all steps are followed correctly, you should see three new Docker containers running. You can check the status of the containers using:
+```
+docker compose ps
+```
+//TODO add the output of the command @pigmej
+
+
+You'll also be able to connect to the node's UI by visiting `https://smesher-alpha.spacemesh.network/` in your browser. This is a PoC of the smesher service UI and is not intended to be a production-ready implementation. It uses the smesher service API directly to interact with the smesher service. It's fully open source and the code can be found [here](https://github.com/spacemeshos/smesher-app).
+//TODO make sure that it actually is opened publicly already @pigmej
```

Comment:
Repo is not public at the moment.

Reply:
Yes that's why I added that todo :) `//TODO make sure that it actually is opened publicly already @pigmej`